openedx/core/lib/xblock_serializer/test_api.py

```diff
@@ -17,8 +17,8 @@
 
 # The expected OLX string for the 'Toy_Videos' sequential in the toy course
 EXPECTED_SEQUENTIAL_OLX = """
-<sequential display_name="Toy Videos" format="Lecture Sequence" url_name="Toy_Videos">
-  <html url_name="secret:toylab" display_name="Toy lab"><![CDATA[
+<sequential display_name="Toy Videos" format="Lecture Sequence" url_name="Toy_Videos" source_key="block-v1:edX+toy+2012_Fall+type@sequential+block@Toy_Videos">
+  <html url_name="secret:toylab" source_key="block-v1:edX+toy+2012_Fall+type@html+block@secret:toylab" display_name="Toy lab"><![CDATA[
```

Comment:
I think we need some tests in this file with `write_source_key=False`

Reply:
The `serialize_modulestore_block_for_learning_core` uses `write_source_key=False` and is tested with `test_html_with_static_asset_learning_core`.

Let me know if you'd like an explicit test here!

Reply:
That's fine, can you maybe just add a comment/docstring mentioning that? So I could grep for `write_source_key=False` in the test file and see where it is.

Reply:
I added one explicit text here: [6f1fecc](https://github.com/openedx/edx-platform/pull/37008/commits/6f1fecc733b91817a613d07c3a8cdf81af224051)